### ISSUE TITLE
FTS: Move from `FTS-1925_token_project` to `testing` repo

### ISSUE THREAD

**Issue by rdimaio** [rucio/containers#320]
From a conversation with Mihai: the branch `testing/FTS-1925_token_project` was merged into `develop`, and RPMs from `develop` are pushed to `testing/`. `testing/<branch>/` repos might disappear once FTS is done with a branch, so we should move to the `testing` repo for more stability (and possibly moving to a stable release if we don't require any `testing`-specific functionality)
